x-pack/solutions/observability/plugins/slo/public/components/slo/purge_instances_confirmation_modal/purge_instances_confirmation_modal.tsx

```diff
@@ -85,13 +95,21 @@ export function PurgeInstancesConfirmationModal({ items, onCancel, onConfirm }:
                 'xpack.slo.purgeInstancesConfirmationModal.descriptionTextWithSelection',
                 {
                   defaultMessage:
-                    'This action will permanently delete stale instances from the {count, plural, one {selected SLO} other {# selected SLOs}} based on the stale duration threshold defined in your settings. You can override this threshold below.',
-                  values: { count: items.length },
+                    'Permanently remove all stale instances from the {count} selected {count, plural, one {SLO} other {SLOs}} based on the {staleSloThresholdLabel} setting. Override this setting by updating the following {staleThresholdLabel}.',
```

Comment:
```suggestion
                    'Permanently delete all stale instances from the {count} selected {count, plural, one {SLO} other {SLOs}} based on the {staleSloThresholdLabel} setting. Override this setting by updating the following {staleThresholdLabel}.',
```
Just to align the two messages. I probably forgot to update this one.